www/js/tablet.js

```diff
@@ -172,96 +167,93 @@ const toggleUnits = () => {
 //   id('jog-distance').value = distance
 // }
 
-const jogTo = (axisAndDistance) => {
-  // Always force G90 mode because synchronization of modal reports is unreliable
-  // JogFeedRate is defined in controls.js
-  let feedrate = JogFeedrate(axisAndDistance);
-  if (modal.units === "G20") {
-    feedrate /= 25.4;
-    feedrate = feedrate.toFixed(2);
-  }
-
-  // tabletShowMessage("JogTo " + cmd);
-  sendCommand(`$J=G91F${feedrate}${axisAndDistance}\n`);
-}
-
 const goAxisByValue = (axis, coordinate) => {
   tabletClick()
   moveTo(axis + coordinate)
 }
 
 const setAxisByValue = (axis, coordinate) => {
   tabletClick();
-  sendCommand(`G10 L20 P0 ${axis}${coordinate}`);
+  const cmd = `G10 L20 P0 ${axis}${coordinate}`;
+  sendCommand(cmd);
 }
 
 const setAxis = (axis, field) => {
   tabletClick();
-  sendCommand(`G10 L20 P1 ${axis}${id(field).value}`);
+  const cmd = `G10 L20 P1 ${axis}${getValue(field)}`;
```

Comment:
Why is this better practice? Not complaining, just want to learn something

Reply:
For us, this makes it easier to insert a `console.info(cmd);` in between the two lines to diagnose what's going on - i.e. our current issue of finding out what is impacting a jog that moves both X & Y.

There's a guiding coding principle (note that is 'guiding' not 'rules') of how to write and structure object orientated code (and JavaScript can pretend to do this)
https://en.wikipedia.org/wiki/SOLID

The 'S', for 'single responsibility principle' is what I'm following here. That one line was doing two different tasks; building up a funky special string, and then sending that string off to some function. By breaking it into two lines that each have only one job, it becomes a lot easier to read and reason on.